python/composio/__init__.py

```diff
@@ -2,17 +2,30 @@
 
```

Comment:
The addition of JSON and Path imports along with the `_hook` function to handle user configuration is a good enhancement. However, ensure that the `api_key` is securely managed and not exposed in any logs or error messages.